roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb

```diff
@@ -8,11 +8,13 @@
   end
 end
 
-files = ['api_paste.ini', 'event_pipeline.yaml', 'policy.json',
-  'ceilometer.conf', 'event_definitions.yaml', 'pipeline.yaml']
-files.each do |file|
+files = {'api_paste.ini'=> 640, 'event_pipeline.yaml'=> 640, 'policy.json'=> 644,
```

Comment:
right, but we should only be checking on it if it's a security issue,  not just a "its probably best if...." which is being set as by ansible anyways.    basically think of serverspec as catching config drift for "stuff that really really matters"